templates.go

```diff
@@ -65,7 +65,8 @@ kubectl config set-cluster {{ .K8SName }} \
 --certificate-authority=$tmp \
 --embed-certs=true
 
-kubectl config set-context {{ .K8SName }} --cluster={{ .K8SName }} --user='{{ .Claims.Email }}'
+kubectl config set-context {{ .K8SName }} --cluster={{ .K8SName }} --user='{{ .Claims.Email }}' --namespace=jobteaser
```

Comment:
I am not sure how handle the namespace. Any ideas ? 😄

Reply:
I wouldn't set namespace here
- override already configured namespace
- which namespace should set by default ?